### ISSUE TITLE
Scope include model

### ISSUE THREAD

**Issue by PatrickGeyer** [RobinBuschmann/sequelize-typescript#601]
**Versions**

- sequelize: 5.6
- sequelize-typescript: 1.0.0-alpha.9
- typescript: 3.4.3

**I'm submitting a ...** 

[X] bug report
[ ] feature request

**Actual behavior:**
When setting a scope for a model, typescript shows the below error.

```
Type '{ model: () => typeof User; as: string; attributes: string[]; include: (() => typeof Image)[]; }' is not assignable to type 'ModelClassGetter'.
              Object literal may only specify known properties, and 'model' does not exist in type 'ModelClassGetter'.
```

**Expected behavior:**
Worked fine before updating to canary release, same code now throwing the above error.
I can still include models if I do as follows: 
```
include: [() => MyModel]
```
but if I need to specify attributes etc, TypeScript will throw error.
**Steps to reproduce:**

**Related code:**

```
@Scopes({
    list: {
        include: [
            {
                model: () => User.scope(null),
                as: 'User',
                attributes: ['firstName', 'lastName', 'id', 'liUserId', 'publicIdentifier'],
                include: [() => Image]
            },
            () => Company
        ]
    },
    worker: {}
})```

**Comment by PatrickGeyer** [comment]
Just discovered that this error is only shown if there is an included model that is included directly with a getter function, not as the ```{model: () => MyModel}``` object.
E.g. the below does NOT throw a TypeError:

```
@Scopes({
    list: {
        include: [
            {
                model: () => User.scope(null),
                as: 'User',
                attributes: ['firstName', 'lastName', 'id', 'liUserId', 'publicIdentifier'],
                include: [() => Image]
            },
            {
                model: () => Company
            }
        ]
    },
    worker: {}
})
```

However, the following fails again if I just add the include key to the included model

```
@Scopes({
    list: {
        include: [
            {
                model: () => User.scope(null),
                as: 'User',
                attributes: ['firstName', 'lastName', 'id', 'liUserId', 'publicIdentifier'],
                include: [() => Image]
            },
            {
                model: () => Company,
                include: []
            }
        ]
    },
    worker: {}
})
```

**Comment by RobinBuschmann** [comment]
@PatrickGeyer Thanks for bringing this up.

**Comment by RobinBuschmann** [comment]
Fix is released: `sequelize-typescript@1.0.0-beta.2`
